click-to-load/click_to_load_config_reference.json

```diff
@@ -0,0 +1,50 @@
+{
+    "Tracker INC": {
+        "domains": [
+            "tracker.test",
+            "tracker.net"
+        ],
+        "excludedSubdomains": [
+            "excluded.tracker.test"
+        ],
+        "excludedDomains": [
+            {
+                "domain": "local-exception.test",
+                "reason": "Local exception"
+            }
+        ],
+        "surrogates": [
+            {
+                "rule": "tracker\\.test\\/[a-z_A-Z]+\\/sdk.js",
+                "surrogate": "surrogate.js",
+                "xray": "surrogate-xray.js"
+            }
+        ],
+        "informationalModal": {
+            "icon": "blocked_logo.svg",
+            "messageTitle": "Logging in with Tracker lets them track you",
+            "messageBody": "Once you're logged in, DuckDuckGo can't block Tracker content from tracking you on this site.",
+            "confirmButtonText": "Log In",
+            "rejectButtonText": "Go back"
+        },
+        "clicksBeforeSimpleVersion": 3,
+        "elementData": {
+            "Page iFrames": {
+                "selectors": [
+                    "iframe[src*='://www.tracker.com/plugins/page']",
+                    "iframe[src*='://www.tracker.com/v2.0/plugins/page']"
+                ],
+                "replaceSettings": {
```

Comment:
I think the `icon` setting is also expected for the `dialog` type. See the logic here[1].

1- https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/d210c7d/shared/js/content-scripts/click-to-load.js#L650

Reply:
I added it, but it looks like our production config is missing it - https://staticcdn.duckduckgo.com/useragents/social_ctp_configuration.json ( see ""infoTitle": "DuckDuckGo blocked this Facebook Page",").